apps/mobile/src/features/Assets/Assets.container.tsx

```diff
@@ -18,5 +22,22 @@ const tabItems = [
 ]
 
 export function AssetsContainer() {
+  const { isAppNotificationEnabled, promptAttempts } = useNotifications()
+  const dispatch = useAppDispatch()
+  const router = useRouter()
+
+  /*
+   * If the user has not enabled notifications and has not been prompted to enable them,
+   * redirect to the opt-in screen
+   * */
+
+  const shouldShowOptIn = !isAppNotificationEnabled && !promptAttempts
+
+  useEffect(() => {
+    if (shouldShowOptIn) {
+      dispatch(updatePromptAttempts(1))
+      router.navigate('/notifications-opt-in')
+    }
+  }, [])
```

Comment:
This is weird. The AssetsContainer should not care about notifications. 
Get some inspiration from here:
https://docs.expo.dev/router/reference/authentication/
https://reactnavigation.org/docs/auth-flow
I think that the _layout.tsx should be the place where we decide what first screen we show to the user.

Reply:
When the user clicks on "get started/continue" in the onboarding flow we should set a flag "onboarding_done": true. Next time the user starts we should not show the onboarding screens. So we will be deciding in the _layout.tsx which screen to show anyway.